### PR TITLE
Headless testing with poltergeist

### PR DESCRIPTION
**Pull request by basiszwo**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://basiszwo-headless-testing.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://basiszwo-headless-testing.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by oliverguenther** [comment]
Just saw this passing on travis. Awesome! :+1:

**Comment by basiszwo** [comment]
still lots of work to do. Currently we are running some in selenium mode but we're getting there :wink:

**Comment by basiszwo** [comment]
@why-el can you please review and merge? I needed to disable poltergeist (again) as we need to update jenkins first. this can be done next week.

**Comment by why-el** [comment]
Ok, on it.

File: script/files/parallel_runtime_rspec.log [hunk 1]
```diff
@@ -0,0 +1,443 @@
+spec/controllers/work_packages/bulk_controller_spec.rb:99.308038
```
Comment (why-el) [inline script/files/parallel_runtime_rspec.log:1]: Is this .log file necessary? Same with the one above.

Comment (basiszwo) [inline script/files/parallel_runtime_rspec.log:1]: I will improve this at a later point to re-write it with every run and download it from somewhere. I have it on my list.

**Pull request merged**